chia/types/coin_spend.py

```diff
@@ -29,3 +30,16 @@ def additions(self) -> List[Coin]:
     # called on untrusted input
     def reserved_fee(self) -> int:
         return fee_for_solution(self.puzzle_reveal, self.solution, INFINITE_COST)
+
+    def hints(self) -> List[bytes]:
```

Comment:
we should rename this to something that makes it clear that it's a slow operation, like `run_clvm_and_get_hints`

Reply:
Also we should probably move this somewhere else

Reply:
agree, I'm not sure if we even use this. @matt-o-how any idea why we added this?